vsc_pdf_template_transformer/lib/utils/evaluator.dart

```diff
@@ -179,3 +185,74 @@ TextOverflow? evaluateTextOverflow(
     dynamic expression, Map<String, dynamic> data) {
   return evaluateEnum(TextOverflow.values, expression, data);
 }
+
+PageOrientation? evaluatePageOrientation(
+    dynamic expression, Map<String, dynamic> data) {
+  return evaluateEnum(PageOrientation.values, expression, data);
+}
+
+PdfPageFormat? evaluatePageFormat(
+    dynamic expression, Map<String, dynamic> data) {
+  final result = _evaluateDynamic(expression, data);
+  if (result == null) {
+    return null;
+  }
+  switch (result) {
+    case 'a3':
+      return PdfPageFormat.a3;
+    case 'a4':
+      return PdfPageFormat.a4;
+    case 'a5':
+      return PdfPageFormat.a5;
+    case 'a6':
+      return PdfPageFormat.a6;
+    case 'letter':
+      return PdfPageFormat.letter;
+    case 'legal':
+      return PdfPageFormat.legal;
+    case 'roll57':
+      return PdfPageFormat.roll57;
+    case 'roll80':
+      return PdfPageFormat.roll80;
+    case 'undefined':
+      return PdfPageFormat.undefined;
+    case 'standard':
+      return PdfPageFormat.standard;
+    default:
+      throw Exception('Invalid page format: $result');
+  }
+}
+
+MainAxisAlignment? evaluateMainAxisAlignment(
+    dynamic expression, Map<String, dynamic> data) {
+  return evaluateEnum(MainAxisAlignment.values, expression, data);
+}
+
+CrossAxisAlignment? evaluateCrossAxisAlignment(
+    dynamic expression, Map<String, dynamic> data) {
+  return evaluateEnum(CrossAxisAlignment.values, expression, data);
+}
+
+List<Widget> getChildren(List<dynamic> children, Map<String, dynamic> data) {
+  final List<Widget> res = [];
+
+  for (final e in children) {
+    if (e['className'] == 'TplRepeater') {
+      final arr = TplRepeater.fromJson(e).toPdf(data);
+      res.addAll(arr);
+    } else {
+      const widgetClassFromJson = {
+        'TplText': TplText.fromJson,
+        'TplSizedBox': TplSizedBox.fromJson,
+        'TplContainer': TplContainer.fromJson,
+        'TplColumn': TplColumn.fromJson,
+        'TplRow': TplRow.fromJson,
+      };
+
+      final fromJson = widgetClassFromJson[e['className']];
+      final widget = fromJson!(e);
```

Comment:
`Transformer.getWidgetBuilder()` does this same thing.  We should just call it from here.